### PR TITLE
Update to snapshot from 2019-05-10 to pick up OpenJDK 11.0.3

### LINKED ISSUE #354: Upgrade to Java 11.0.3 and close CVE-2019-2602
Please release an update on gcr.io/distroless-java:11 to upgrade to 11.0.3. There is a CVE out affecting 11.0.2.

### PR DESCRIPTION
**Pull request by briandealwis**
Fixes #354

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -96,9 +96,7 @@ dpkg_list(
         "python2.7-minimal",
         "libpython2.7-stdlib",
         "dash",
-        # Version required to skip a security fix to the pre-release library
-        # TODO: Remove when there is a security fix or dpkg_list finds the recent version
-        "libc-bin=2.24-11+deb9u4",
+        "libc-bin",
```
Comment (chanseokoh) [inline WORKSPACE]: Are we good now? Should we also update `"libc6=2.24-11+deb9u4"` above?

Comment (briandealwis) [reply WORKSPACE]: Trying to figure out which packages are resolved is tricky.  Turns out we're still pulling down `2.24-1deb9u1`, not `u4`.  I'll restore libc-bin.

Comment (briandealwis) [reply WORKSPACE]: The easiest way I've found to discover package versions is to look in `bazel-xxx/external/package_bundle/file/packages.bzl`

Comment (briandealwis) [reply WORKSPACE]: Ah can also look at the contents of the files in `/var/lib/dpkg/status.d/` within the image.

**Pull request merged**